src/components/call-to-action.tsx

```
@@ -101,14 +104,17 @@ function CtaText() {
 }
 
 export default function CallToAction() {
-  let prefersReducedMotion = useReducedMotion();
+  let prefersReducedMotion = false;
```

ESLint

[error] 'prefersReducedMotion' is never reassigned. Use 'const' instead.

(prefer-const)